aegisthus-hadoop/src/main/java/com/netflix/aegisthus/util/ObservableToIterator.java

```diff
@@ -57,7 +62,9 @@ public boolean hasNext() {
                     buf = take();
                 }
                 if (buf.isOnError()) {
-                    throw Exceptions.propagate(buf.getThrowable());
```

Comment:
This is some stacktrace.
```
2015-03-30 23:48:09,163 ERROR [main] com.netflix.aegisthus.util.ObservableToIterator: Meet buffer error due to:
java.io.IOException: Corrupt (negative) value length encountered
        at org.apache.cassandra.utils.ByteBufferUtil.readWithLength(ByteBufferUtil.java:352)
        at org.apache.cassandra.db.ColumnSerializer.deserializeColumnBody(ColumnSerializer.java:118)
```

Do you mean I can change to `buf. getThrowable().getMessage()` ?